src/components/richLink.tsx

```diff
@@ -0,0 +1,165 @@
+import type { SerializedStyles } from '@emotion/core';
+import { css, jsx as styledH } from '@emotion/core';
+import { remSpace } from '@guardian/src-foundations';
+import { from } from '@guardian/src-foundations/mq';
+import { neutral } from '@guardian/src-foundations/palette';
+import { headline, textSans } from '@guardian/src-foundations/typography';
+import { SvgArrowRightStraight } from '@guardian/src-icons';
+import { Design, Pillar } from '@guardian/types/Format';
+import type { Format } from '@guardian/types/Format';
+import { createElement as h } from 'react';
+import type { ReactElement } from 'react';
+import { basePx, darkModeCss, darkModeStyles } from 'styles';
+import { getThemeStyles } from 'themeStyles';
+
+export const richLinkWidth = '8.75rem';
+
+const richLinkPillarStyles = (kicker: string): string => {
+	return `
+		border-top: solid 1px ${kicker};
+
+		${darkModeStyles`
+			border-top: solid 1px ${neutral[60]};
+		`}
+
+		svg {
+			fill: white;
+			background: ${kicker};
+			border-color: ${kicker};
+			${darkModeStyles`
+				border-color: ${neutral[60]};
+				background: ${neutral[20]};
+				fill: ${neutral[60]};
+			`}
+		}
+
+		button {
+			color: ${kicker};
+			${darkModeStyles`
+				color: ${neutral[60]};
+			`}
+		}
+	`;
+};
+
+const richLinkStyles = (format: Format): SerializedStyles => {
+	const backgroundColor =
+		format.design === Design.Comment ? neutral[86] : neutral[97];
+
+	const { kicker: newsKicker } = getThemeStyles(Pillar.News);
+	const { kicker: opinionKicker } = getThemeStyles(Pillar.Opinion);
+	const { kicker: sportKicker } = getThemeStyles(Pillar.Sport);
+	const { kicker: cultureKicker } = getThemeStyles(Pillar.Culture);
+	const { kicker: lifestyleKicker } = getThemeStyles(Pillar.Lifestyle);
```

Comment:
updated from getPillar?

Reply:
yep we renamed `getPillar` to `getTheme` recently